scripts/open_targets_biocypher_run.py

```diff
@@ -2,12 +2,16 @@
 
 # VSCode does not add the root directory to the path (by default?). Not sure why
 # this works sometimes and not others. This is a workaround.
-import sys
-from pyspark import StorageLevel
 
-sys.path.append("")
+# TODO: Delete lines 3,4, 10 and 13. By adding line 6 in pyptoject.toml, and installing the project using 
```

Comment:
As my another comment mentioned, I think we can simply remove the workaround here and the relevant comment block.

Reply:
Could be, I haven't tested that recently.

Reply:
Resolving this conversation: I've removed the comments because we now know what was going on.